tools/src/tester/types/story.types.ts

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable */
 
+import { ChapterOutput } from "./eval.types";
+
```

Comment:
This file is not supposed to be edited by hand. It's generated from running `npx ts-node tools/src/tester/_generate_story_types.ts`.

The goal is maintaining parity between the JSON Schemas defined in `test_story.schema.yaml` and the TS types defined here.